src/Backend/Doctrine/Entity/CreateSchema.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Backend\Doctrine\Entity;
+
+use Doctrine\DBAL\Exception\TableExistsException;
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\Tools\SchemaTool;
+
+class CreateSchema
+{
+    /** @var EntityManager */
+    private $entityManager;
+
+    /**
+     * @param EntityManager $entityManager
+     */
+    public function __construct(EntityManager $entityManager)
+    {
+        $this->entityManager = $entityManager;
+    }
+
+    /**
+     * Adds a new doctrine entity in the database
+     *
+     * @param string $entityClass
+     */
+    public function forEntityClass($entityClass)
+    {
+        $this->forEntityClasses([$entityClass]);
+    }
+
+    /**
+     * Adds new doctrine entities in the database
+     *
+     * @param array $entityClasses
+     */
+    public function forEntityClasses(array $entityClasses)
+    {
+        // create the database table for the given class using the doctrine SchemaTool
+        $schemaTool = new SchemaTool($this->entityManager);
+
+        try {
+            $schemaTool->createSchema(
+                array_map(
+                    function ($entityClass) {
+                        return $this->entityManager->getClassMetadata($entityClass);
+                    },
+                    $entityClasses
+                )
```

Comment:
I think you can change your array map to this:

```php
array_map(
    array($this->entityManager, 'getClassMetadata'),
    $entityClasses
)
```

By using a callable, you don't need the anonymous function just wrapping the method call.